src/Apps/Collect2/Routes/Collection/index.tsx

```diff
@@ -74,13 +75,16 @@ export class CollectionApp extends Component<CollectionAppProps> {
       : `Buy, bid, and inquire on ${title} on Artsy.`
 
     const showCollectionHubs = collection.linkedCollections.length > 0
+
+    const socialImage =
+      headerImage || fallbackHeaderImage?.edges[0]?.node?.image?.resized.url
```

Comment:
I think this line will still fail if `edges` returns `null`-- but a test can confirm!